_inc/client/setup-wizard/intro-page/index.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { imagePath } from 'constants/urls';
+
+import './style.scss';
+
+const IntroPage = props => {
+	return (
+		<div className="jp-setup-wizard-intro">
+			<img
+				width="200px"
+				height="200px"
+				src={ imagePath + '/jetpack-powering-up.svg' }
+				alt={ __( 'A jetpack site powering up' ) }
+			/>
+			<h1 className="jp-setup-wizard-header">
+				{ __( 'Set up Jetpack for better site security, performance, and more' ) }
+			</h1>
+			<p className="jp-setup-wizard-paragraph">
+				{ __(
+					'Jetpack is a cloud-powered tool built by Automattic and brought to you by Bluehost.'
+				) }
+			</p>
+			<p className="jp-setup-wizard-paragraph">
+				{ __(
+					'Answer a few questions and we’ll help you secure, speed up, customize, and grow your WordPress website.'
+				) }
+			</p>
+			<div className="jp-setup-wizard-intro-question">
+				<h2>
+					{ __( 'What will %(siteUrl)s be used for?', { args: { siteUrl: props.siteRawUrl } } ) }
```

Comment:
I don't know if the raw site URL is the best thing to use here. It looks okay for some sites, but as soon as the site is in a subdirectory for example, it will look like this:
![image](https://user-images.githubusercontent.com/426388/79876623-934be800-83eb-11ea-87e2-22ec5ba561f4.png)

Could we use the site title instead?

Reply:
Yes I think that's fine so I went ahead and added it. CC @jeffgolenski to be sure. This change means the above example would display as "What will Inexpensive Teira be used for?".